homeassistant/components/system_bridge/config_flow.py

```diff
@@ -51,17 +50,33 @@ async def _validate_input(
     websocket_client = WebSocketClient(
         host,
         data[CONF_PORT],
-        data[CONF_API_KEY],
+        data[CONF_TOKEN],
     )
+
+    system_data: System | None = None
+
+    async def async_handle_module(
+        module_name: str,
+        module: Any,
+    ) -> None:
+        """Handle data from the WebSocket client."""
+        _LOGGER.debug("Set new data for: %s", module_name)
+        if module_name == "system":
+            nonlocal system_data
```

Comment:
Instead of using a nonlocal, can we move this function into the config flow class?